test/blackbox-tests/test-cases/pkg/convert-opam-commands.t

```diff
@@ -184,54 +184,54 @@ Package which has boolean where string was expected. This should be caught while
    (progn
     (when
      %{pkg-self:foo}
-     (run echo a))
+     (system "echo a"))
```

Comment:
For example, we could improve an echo step and translate it into the echo action.